### ISSUE TITLE
Jump to translation key in the source code.

### ISSUE THREAD

**Issue by giolf** [marhali/easy-i18n#161]
It would be great to have the possibility to click in the key locale file to jump in the source code on which that key is used.

For example if i have:
```
// en.json
{
  "test": "test EN"
}
```
when i click on `test` key, it would be great to jump in the source code where that key is used:
```
// App.vue

<template>
 <div class="wrapper">
   {{ $t('test') }} <---| jump here
 </div>
</template>
```

**Comment by marhali** [comment]
Isn't this something like the existing `Find Usages` functionality?

**Comment by giolf** [comment]
> Isn't this something like the existing `Find Usages` functionality?

Yes… is similar. The power of what I mean is that as soon as you click to the key you jump to the source! I’m quite sure that is possible to do it because I saw other plug-in that are doing that. 

If I use the the `Find usages` Feature, it would not be very useful because you are looking for a key like `title` But in real, you should use the entire path of the key (nested namespace). That is why it would make sense to implement it in the plug-in.

what do you think ?
